README.md

```diff
@@ -1 +1 @@
-"# GithubSplunk" 
+This is just a github test project. This doesn't meant to be any serious project. It's may be not even in a correct structure. My intention is to make it simple a possible.
```

Comment:
I think we can improve this description a little.

Reply:
Okay, I will submit the correction soon as I can.